links.json

```diff
@@ -0,0 +1,10 @@
+{
+    "models": [
+        "https://huggingface.co/NoCrypt/expermental_models/resolve/main/shux7.safetensors:enabled"
+    ],
+    "vae": [
+        "https://huggingface.co/NoCrypt/resources/resolve/main/VAE/any.vae.safetensors:enabled"
```

Comment:
i dunno if appending `:enabled` and `:keep` is a good idea in term of UX

Reply:
Alright I will move them into separate arrays then, something like this :
`{
    "models": [
        "https://huggingface.co/NoCrypt/expermental_models/resolve/main/shux7.safetensors"
    ],
...
    "link_to_name": [],
    "enabled": ["https://huggingface.co/NoCrypt/expermental_models/resolve/main/shux7.safetensors"],
    "keep": []
}`

Reply:
I made the change on the last commit 24e1e4d